src/lib/journey/journey.store.ts

```diff
@@ -75,6 +83,10 @@ export function initialize(initOptions?: StepOptions): JourneyStore {
   let stepNumber = 0;
 
   async function next(prevStep: StepTypes = null, nextOptions?: StepOptions, resumeUrl?: string) {
+    if (!Config.get().serverConfig?.baseUrl) {
```

Comment:
This will help us catch and report issues of misconfiguration or no configuration.